queue-event-handler.js

```diff
@@ -39,20 +39,24 @@ const addItem = ({ queue }) => {
         .catch(logger.error);
 };
 
-const removeItem = ({ queue, item }) => {
+const removeItem = ({ queue, item, meta }) => {
```

Comment:
You should (have) default `meta` to `{}` to avoid errors if it's not available (e.g. https://github.com/branch-bookkeeper/ugo/blob/master/routes-queue.js#L30-L30)